tools/format.py

```diff
@@ -48,24 +48,15 @@ def main():
         did_fmt = True
 
     if not did_fmt:
-        prettier()
+        dprint()
         yapf()
         rustfmt()
 
 
-def prettier():
-    script = os.path.join(third_party_path, "node_modules", "prettier",
-                          "bin-prettier.js")
-    source_files = get_sources(root_path, ["*.js", "*.json", "*.ts", "*.md"])
-    if source_files:
-        max_command_length = 24000
-        while len(source_files) > 0:
-            command = ["node", script, "--write", "--loglevel=error", "--"]
-            while len(source_files) > 0:
-                command.append(source_files.pop())
-                if len(" ".join(command)) > max_command_length:
-                    break
-            run(command, shell=False, quiet=True)
```

Comment:
If we were to maintain this functionality, then I think this code would have to know which files are ignored/excluded. See my comments here and let me know if I'm wrong: https://github.com/dprint/dprint/issues/258

Reply:
Also, see my comments here about possibly adding "--git-staged" to dprint: https://github.com/dprint/dprint/issues/255 (don't want to do it)

I think perhaps we should consider removing this git staged functionality from this script? It runs decently fast now as-is.